src/ContractEncoder.js

```diff
@@ -1,13 +1,13 @@
-const RLP = require('rlp')
 const ApiEncoder = require('./ApiEncoder')
 const Serializer = require('./Serializer')
-const BytecodeSerializer = require('./Serializers/BytecodeSerializer')
-const {byteArray2Int, byteArray2Hex} = require('./utils/int2ByteArray')
+const ContractBytecodeSerializer = require('./Serializers/ContractBytecodeSerializer')
```

Comment:
I would name it `ContractBytecodeSerializer` but it is already taken for contract address serializer.

Reply:
I don't get the comment, the name is `ContractBytecodeSerializer` ?

Reply:
I'm sorry, I would name it `ContractSerializer`